google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -118,8 +118,8 @@ public void connect(@NotNull ConnectionCallback<AppEngineDeploymentConfiguration
 
       if (!Services.getLoginService().isLoggedIn()) {
         callback.errorOccurred(GctBundle.message("appengine.deployment.error.not.logged.in"));
-      } else if (CloudSdkUtil.isCloudSdkExecutable(
-          CloudSdkUtil.toExecutablePath(CloudSdkService.getInstance().getCloudSdkHomePath()))) {
+      } else if (CloudSdkUtil.isCloudSdkExecutable(CloudSdkUtil.toExecutablePath(
+          CloudSdkService.getInstance().getCloudSdkHomePath().toString()))) {
```

Comment:
Also, CloudSdkPanel does not force the user to have a non-empty or valid SDK. Its up to users of the panel (other panels that include it) decide if they want to override this behavior.